front/src/routes/integration/all/yeelight/YeelightPage.jsx

```diff
@@ -0,0 +1,61 @@
+import { Text } from 'preact-i18n';
+import { Link } from 'preact-router/match';
+import DeviceConfigurationLink from '../../../../components/documentation/DeviceConfigurationLink';
+
+const YeelightPage = ({ children, user }) => (
+  <div class="page">
+    <div class="page-main">
+      <div class="my-3 my-md-5">
+        <div class="container">
+          <div class="row">
+            <div class="col-lg-3">
+              <h3 class="page-title mb-5">
+                <Text id="integration.yeelight.title" />
+              </h3>
+              <div>
+                <div class="list-group list-group-transparent mb-0">
+                  <Link
+                    href="/dashboard/integration/device/yeelight"
+                    activeClassName="active"
+                    class="list-group-item list-group-item-action d-flex align-items-center"
+                  >
+                    <span class="icon mr-3">
+                      <i class="fe fe-link" />
+                    </span>
+                    <Text id="integration.yeelight.deviceTab" />
+                  </Link>
+
+                  <Link
+                    href="/dashboard/integration/device/yeelight/discover"
+                    activeClassName="active"
+                    class="list-group-item list-group-item-action d-flex align-items-center"
+                  >
+                    <span class="icon mr-3">
+                      <i class="fe fe-radio" />
+                    </span>
+                    <Text id="integration.yeelight.discoverTab" />
+                  </Link>
+
+                  <DeviceConfigurationLink
+                    user={user}
+                    documentKey="yeelight-lan-control"
```

Comment:
This don't match the id you chose in the documentation PR! It should be "yeelight".